easy_phi/auth.py

```diff
@@ -262,24 +267,143 @@ def get(self):
 
     post = get
 
+_user_cache = None
+
+
+class PasswordAuthAPIHandler(tornado.web.RequestHandler):
+    user = None
```

Comment:
This handler implements both api and page rendering. Later page rendering will be moved to admin console